osquery/tables/networking/listening_ports.cpp

```diff
@@ -14,47 +14,41 @@
 namespace osquery {
 namespace tables {
 
-typedef std::pair<std::string, std::string> ProtoFamilyPair;
-typedef std::map<std::string, std::vector<ProtoFamilyPair>> PortMap;
-
 QueryData genListeningPorts(QueryContext& context) {
   QueryData results;
 
   auto sockets = SQL::selectAllFrom("process_open_sockets");
 
-  PortMap ports;
   for (const auto& socket : sockets) {
-    if (socket.at("remote_port") != "0") {
-      // Listening UDP/TCP ports have a remote_port == "0"
+    // 1  = AF_UNIX
+    if (socket.at("family") == "1" && socket.at("path").empty()) {
```

Comment:
Maybe define "1" "2" and "10" as constants somewhere.

Reply:
This is also code I took as it was; but adding a couple of constants is super easy, so we can do it right now

I also had problems understanding what each value was initally, so I added some comments above each usage. I'm not sure why it wasn't done before!

Reply:
Perfect! Thanks.